demo/terraform/demo_stack/stack.tf

```diff
@@ -96,6 +96,6 @@ module "stack" {
 
   app_containers = {
     container_registry = "public.ecr.aws/y1p3h6z3"
-    container_tag      = "latest"
+    container_tag      = "ref.c3b7e5a1557ba0115a31ad06e6a7edd5ad143a64"
```

Comment:
Similar to above can we tag a particular container image to denote that it is the demo image?

Reply:
It's not really "demo" in this case 🤔